vagga.yaml

```diff
@@ -121,6 +121,7 @@ commands:
   test: !Command
     description: Run self tests
     container: test
+    prerequisites: [make]
```

Comment:
This one doesn't work. And is explicitly [documented as such](http://vagga.readthedocs.io/en/latest/commandline.html#multiple-commands)